### ISSUE TITLE
can use for react native 0.26 ?

### ISSUE THREAD

**Issue by tdhungit** [asamiller/react-native-local-image-manager#2]
Please help me play on React Native 0.26. Thanks

**Comment by shaneosullivan** [comment]
Yes please! With iOS 10 coming, I hear RN is busted without a recent fix, so if you could ensure this is working with the latest version it'll really help
